strongbox-parent/pom.xml

```diff
@@ -69,7 +69,8 @@
         <version.jetty>9.4.6.v20170531</version.jetty>
         <version.lucene>4.8.1</version.lucene>
         <version.logback>1.2.3</version.logback>
-        <version.maven>3.3.9</version.maven>
+        <version.maven>3.6.0</version.maven>
+        <version.maven.strongbox.fork>3.6.1-PR-2-SNAPSHOT</version.maven.strongbox.fork>
```

Comment:
So, are we forking Maven for it's metadata API, or not? (Is this is a leftover...?).

Reply:
yes, see https://github.com/strongbox/maven/pull/2